.circleci/config.yml

```diff
@@ -150,7 +150,25 @@ commands:
   provision-hardhat-with-packaged-solcjs:
     description: "Clones Hardhat repository and configures it to use a local clone of solc-js."
     steps:
-      - run: git clone --depth 1 "https://github.com/nomiclabs/hardhat" hardhat/
```

Comment:
We also have this: https://github.com/ethereum/solc-js/blob/2f4e680edc2a89ead36d8b1ec4ee637a95111203/.circleci/config.yml#L341

It's already pulling in a release, but there's technically a small risk that their releases on github and npm are not in sync. I would not expect that to happen often but if it happens it would be a hell to debug this.

Overall, it would be better to have a separate task that just fetches the versionn number of the latest release and stores it in a variable. Then all other tasks could just use that number.

Reply:
Right. But I believe that you cannot access environment variables defined in one job in another job, but we could store the version in the persistent storage and load in another jobs. Otherwise, the option would be to fetch the version at each job, which is what this currently do.

But yeah, the `npm update` would need to install the same version. I will fix that.

Reply:
I didn't mean to actually share the value jobs. What I meant was a shared *step*, like for example `show-npm-version`, which the job that needs the value can run and the value gets added to its own env.

Anyway, I wanted to suggest that as a potential improvement, but it should be done separately, if at all. This PR is fine to merge without it.

Reply:
Oh, I just did the change since it was not much different of what the PR was doing anyway.